.github/workflows/build.yml

```diff
@@ -248,7 +247,6 @@ jobs:
         # Render analyzer waveform tests to an offscreen buffer
         QT_QPA_PLATFORM: ${{ matrix.qt_qpa_platform }}
         GTEST_COLOR: 1
-        CTEST_PARALLEL_LEVEL: 2
```

Comment:
Revert this

Reply:
I will before merging. I just didn't want to wait another 35 minutes for a build to fail again.